C/ConnectFourSolver/build_tarballs.jl

```diff
@@ -1,44 +1,43 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
+import BinaryBuilderBase: nbits
```

Comment:
@giordano perhaps BB should re-export `nbits`, just like it does for `arch`, `libc` and other similar functions?

Reply:
Sounds good!